docs/cli/defradb_client_peerid.md

```diff
@@ -1,6 +1,10 @@
 ## defradb client peerid
 
-Get the PeerID of the DefraDB node
+Get the PeerID of the node
```

Comment:
Thought: should this be `DefraDB node` instead of `node` now?

Reply:
I don't ming going with either of them. Do you have a strong preference?